htdocs/contact/class/contact.class.php

```diff
@@ -739,6 +739,7 @@ public function fetch($id, $user = null, $ref_ext = '', $email = '')
 				$this->ref_ext			= $obj->ref_ext;
 
 				$this->civility_code    = $obj->civility_code;
+				$this->civility_id      = $obj->civility_code;
```

Comment:
Why do you need this ?
We try to get into id, the value of id and into code the value of code.

Reply:
seems to work now, i close